core/src/main/scala/cats/syntax/option.scala

```diff
@@ -0,0 +1,20 @@
+package cats
+package syntax
+
+import cats.data.Xor
+
+trait OptionSyntax {
+  def none[A] = Option.empty[A]
+  implicit def toOptionSyntax[A](a: A): ToOptionOps[A] = new ToOptionOps(a)
+  implicit def optionSyntax[A](a: Option[A]): OptionOps[A] = new OptionOps(a)
+}
+
+class ToOptionOps[A](val a: A) extends AnyVal {
```

Comment:
This makes me think [`EitherOps`](https://github.com/non/cats/blob/d128c060b92751179276882aa375096cbad209b2/core/src/main/scala/cats/syntax/xor.scala#L10) should be `ToEitherOps` for consistency.